src/pages/rekey-reason/rekey-reason.scss

```diff
@@ -1,4 +1,67 @@
 page-rekey-reason {
+
+  .transferInfo {
+    color: map-get($colors-gds, gds-grey-1);
+    font-weight: initial;
+    letter-spacing: initial;
+  }
+
+  .staffNumber {
+    max-width: 40%;
+    width: 40%;
+    min-width: 40%;
+  }
+
+  .staffNumberRow {
+    height: 70px;
+    max-height: 70px;
```

Comment:
Is there a way to style this without specifying height? Font sizes can increase and this might break?